src/Microsoft.OData.Client/DataServiceClientFormat.cs

```diff
@@ -275,4 +283,89 @@ private static string ChooseMediaType(bool hasSelectQueryOption)
             return MimeApplicationJsonODataLight;
         }
     }
+
+    /// <summary>
+    ///This class is a utility to help with POCO loading it can be registered as an option to load the CSDL over network or from a file or string
+    ///
+    /// </summary>
+    public class DataServiceClientFormatUtil
+    {
+        public IEdmModel EdmModel { get; private set; }
+        /// <summary>
+        /// Calling this initializes the parsing in the constructor
+        /// </summary>
+        /// <param name="metadataUri"></param>
+        public DataServiceClientFormatUtil(Uri metadataUri)
+        {
+            EdmModel = ReadModelFromNetwork(metadataUri, new Dictionary<string, string>());

```

Comment:
Call the second constructor directly

Reply:
Updated to remove the class and have a method only